src/jiraCommon.gs

```diff
@@ -616,7 +614,7 @@ function headerNames(header) {
 }
 
 /**
- * @TODO: not used yet / to be used in convertEpicCell() and further places
+ * Fetches a JIRA issue from the REST API
```

Comment:
This is now used as part of the code to edit (transition) the status of an issue. I found a small bug too if fields was empty it should not be included in the POST to the JIRA API otherwise JIRA will respond with an error